tsconfig.json

```diff
@@ -1,26 +1,23 @@
 {
-  "compilerOptions": {
-    "target": "es5",
-    "lib": [
-      "dom",
-      "dom.iterable",
-      "esnext"
-    ],
-    "allowJs": true,
-    "skipLibCheck": true,
-    "esModuleInterop": true,
-    "allowSyntheticDefaultImports": true,
-    "strict": true,
-    "forceConsistentCasingInFileNames": true,
-    "noFallthroughCasesInSwitch": true,
-    "module": "esnext",
-    "moduleResolution": "node",
-    "resolveJsonModule": true,
-    "isolatedModules": true,
-    "noEmit": true,
-    "jsx": "react-jsx"
-  },
-  "include": [
-    "src"
-  ]
+    "compilerOptions": {
+        "target": "es2022",
+        "lib": ["es2022", "dom"],
+        "module": "node16",
+        "moduleResolution": "node16",
+        "jsx": "react-jsx",
+        "types": [],
+        "sourceMap": true,
+        "outDir": "./dist",
+        "importsNotUsedAsValues": "error",
+        "newLine": "lf",
+        "esModuleInterop": true,
+        "forceConsistentCasingInFileNames": true,
+        "strict": true,
+        "noImplicitReturns": true,
+        "noFallthroughCasesInSwitch": true,
+        "noUncheckedIndexedAccess": true,
+        "noImplicitOverride": true,
+        "skipDefaultLibCheck": true,
+        "noEmit": true
+    }
 }
```

Comment:
why

Reply:
From the doc: [`tsconfig.json`](https://docs.google.com/document/d/1TroPGXxQYTDQeVUFC7LfVZu0iwZHgQegRPATTWOhsBo/edit#heading=h.c8yb07nmvpow)